### PR TITLE
some nice things we thought of, but that were not in stories

### PR DESCRIPTION
**Pull request by ebreton**
**high level:**
1. added command `wp-version`
1. added command `wp-admins`
1. added command `inventory`
1. improved command `check-one`: actually checks that config is ok with wp-cli
1. improved command `generate-one` with parameter `--admin-password` to force password for admin instead of creating a random one
1. improved CLI messages when running jahia2wp.py script

**low level:**
1. improved commands (`run_command`, `run_mysql`, `run_wp_cli`), which actually return output, do not display `stderr`, but keep it available on error cases
1. extended `Utils` to read csv from strings and to provide `run_command`
1. improved `WPSite` with a factory method which take openshift_env and path instead of url
1. improved model `WPUser` to get role
1. improved `WPRawConfig` which now gives access to wp-config variables, and users
openshift_env and url
1. used `veritas.validators` in wordpress models

### CODE REVIEW

File: src/wordpress/configurator.py [hunk 1]
```diff
@@ -18,41 +19,54 @@ class WPRawConfig:
...
+
+        # set initial path
+        given_path = os.path.abspath(path)
+        logging.debug('walking through %s', given_path)
+
+        # walk through all subdirs of given path
+        # topdown is true in order modify the dirnames list in-place (and exclude WP dirs)
+        for (parent_path, dir_names, filenames) in os.walk(given_path, topdown=True):
+            # only keep potential WP sites
+            dir_names[:] = [d for d in dir_names if keep_wp_sites(d)]
```
Comment (GregLeBarbar) [inline src/wordpress/configurator.py]: dir_names[:] = [d for d in dir_names if keep_wp_sites(d)]

Tu veux que toutes les références au contenu de dir_names soient conservées ?

**Comment by GregLeBarbar** [comment]
Une petite PR :-) 
LGTM

File: src/wordpress/configurator.py (hunk 1 above)
Comment (ebreton) [reply src/wordpress/configurator.py]: je veux supprimer du dir_names les répertoires worpdress, ainsi que tous leurs sous-repertoires. cette ligne va avec le commentaire sur topdown.

plus clair ?

**Comment by ebreton** [comment]
> Une petite PR :-)
> LGTM

Bon, elle a pris un peu de poids pendant le week-end, c'est vrai ;)

@lboatto tu as pu revoir ? c'est looks-good-to-me aussi ?  ;)

**Pull request merged**